docker-compose.yml

```diff
@@ -33,3 +33,22 @@ db:
     - POSTGRES_DB=uber_db
 redis:
   image: redis
+rabbitmq:
+  container_name: rams_rabbit
+  image: rabbitmq:alpine
+  environment:
+    - RABBITMQ_DEFAULT_USER=celery
+    - RABBITMQ_DEFAULT_PASS=celery
```

Comment:
I assume we will want to do something more robust about this password before we go to prod.

Reply:
Given that this file is checked in, you may want to have a prod-only script that changes those for the container using docker exec.

Reply:
Ja, something like that. It's fine for where it is: Quickstart isn't really intended to be prod-quality w/out any changes.